nixui/graphics/richtext.py

```diff
@@ -71,3 +73,12 @@ def docbook_to_html(docbook_str):
         return pypandoc.convert_text(docbook_str, 'html', format='docbook')
     except RuntimeError:
         return None  # handle case - pandoc errors due to invalid XML
+
+def example_to_html(example):
+    """
+    Examples can use `literalExample` or `literalExpression` fields for raw text so we can extract that here.
+    """
+    if isinstance(example, dict):
+        if example.get("_type") in ["literalExample", "literalExpression"] and "text" in example:
```

Comment:
nit: If `example.get("_type") in ["literalExample", "literalExpression"]` then `"text" in example` is always true.

Reply:
Ah yep, fixed now. Thanks!